lib/rdoc/parser/rbs.rb

```diff
@@ -0,0 +1,175 @@
+# frozen_string_literal: true
+
+require 'rbs'
+
+module RDoc
+  class Parser
+    class RBS < Parser
+      parse_files_matching(/\.rbs$/)
+
+      def scan
+        ::RBS::RDocPlugin::Parser.new(@top_level, @content).scan
+      end
+    end
+  end
+end
+
+module RBS
+  module RDocPlugin
+    class Parser
```

Comment:
Rename the file to `lib/rbs/rdoc_plugin/parser.rb` to align the class name and the path.